bin/build-hdfs

```diff
@@ -10,86 +16,117 @@ IMPALA_VER=cdh5.3.1-release
 IMPALA_ZIP=$IMPALA_VER.zip
 IMPALA_URL=https://github.com/cloudera/Impala/archive/$IMPALA_ZIP
 NATIVE=native
-DIST=hdfs-mesos-0.1.1
+
+# the full distro is in the $DIST dir or $DIST.tzg
+DIST=hdfs-mesos-$VERSION
+EXECUTOR=hdfs-mesos-executor-$VERSION
 
 # Remove cached binaries and exit
 if [ "$1" == "clean" ]; then
-    rm -f $HADOOP_ZIP
-    rm -rf $HADOOP_DIR
-    rm -rf $NATIVE
-    rm -rf $DIST
-    rm -f $DIST.tgz
+    rm -Rf $BUILD_DIR
     $PROJ_DIR/gradlew clean
```

Comment:
Why are we using -Rf instead of -rf here? It should have the same functionality, just curious.